plugins/interactive.yaml

```
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: interactive
+spec:
+  version: v1.0.3
+  homepage: https://github.com/similarweb/kubectl-interactive
+  shortDescription: A kubectl plugin that provides an interactive wrapper on top of kubectl.
+  description: |
+    A kubectl plugin that provides an interactive wrapper on top of kubectl.
+    This plugin is similar to a proxy in front of kubectl that provides an easier method to use and explore kubernetes objects with confidence.
```

```suggestion
    This plugin is similar to a proxy in front of kubectl that provides an easier method to use and explore kubernetes objects.
```
Please try to explain what your plugin does instead of making it sound like a marketing pitch.

What about this:

> This plugin is a smart wrapper around kubectl, which lets you pick the resource to act on. For example, `kubectl interactive exec` will show a list of pods and execute `kubectl exec <selected-pod>`.

---

Nice. thanks for that.
the text was changed